### PR TITLE
Update Index “index”

### PR DESCRIPTION
**Pull request by james**
Automatically generated by Netlify CMS

### CODE REVIEW

File: index.markdown [hunk 1]
```diff
@@ -1259,6 +1259,95 @@ passing of their probationary period.
...
+    per year to help them with the additional costs of living in London.
+
+* This applies to people who live in a London borough.
+* If someone’s salary increases above this level, London weighting will
+  gradually taper off (by £500 for each £1000 of extra salary) between £40,000
+  and £48,000
+* The rationale for these numbers is taken from Trust for London research which
+  shows London weighting materially benefits people on salaries under £40,000.
+* It is subject to tax & NI and is not pensionable
+* It is a fixed rate, not uplifted by CPI
```
Comment (erbridge) [inline index.markdown:1335]: These bullets should be intented to be subpoints of the numbered list.

Comment (james) [reply index.markdown]: Good catch!

**Review by erbridge — APPROVED** [review]
The inconsistent indentation is bothering me, but it looks like it compiles fine, so it's probably not important.

**Comment by james** [comment]
ah yes, that one that was out. Fixed it!

**Pull request merged**